src/vstest.console/Processors/DisableAutoFakesArgumentProcessor.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.CommandLine.Processors
+{
+    using System;
+    using System.Globalization;
+
+    using CommandLineResources = Microsoft.VisualStudio.TestPlatform.CommandLine.Resources.Resources;
+
+    /// <summary>
+    /// An argument processor that allows the user to disable fakes
+    /// from the command line using the --DisableAutoFakes|/DisableAutoFakes command line switch.
+    /// </summary>
+    internal class DisableAutoFakesArgumentProcessor : IArgumentProcessor
```

Comment:
This is interesting. For `code coverage` we require users to specify `/collect`, for fakes we're automatically trying to figure if fakes should be enabled from the provided test sources. I believe this is for compatibility with TPv1.

cc: @Faizan2304 @pvlakshm fyi